docs/01_key-concepts/05_ram.md

```diff
@@ -43,26 +43,26 @@ The quickest way to calculate the price of RAM:
             "weight": "0.50000000000000000"
         },
         "quote": {
-            "balance": "3158350.8754 EOS",
+            "balance": "3158350.8754 XYZ",
             "weight": "0.50000000000000000"
         }
     }
     ```
 
 3. Make note of the `base balance`, in this case it is 35044821247.
 4. Make note of the `quote balance`, in this case it is 3158350.8754.
-5. Calculate the price of 1Kib of RAM as `quote balance` * 1024 / `base balance` = 0.0922 EOS.
+5. Calculate the price of 1Kib of RAM as `quote balance` * 1024 / `base balance` = 0.0922 XYZ.
 
 ### Buy RAM With Command Line Interface
 
-You can buy RAM through the dune command line interface tool. You can buy either an explicit amount of RAM expressed in bytes or an amount of RAM worth an explicit amount of EOS.
+You can buy RAM through the dune command line interface tool. You can buy either an explicit amount of RAM expressed in bytes or an amount of RAM worth an explicit amount of Vaulta.
 
-### Buy RAM In EOS
+### Buy RAM In XYZ
 
-For example, the command below buys for account `bob` 0.1 EOS worth of RAM at the current market RAM price. The cost for the RAM and the execution of this transaction is covered by the `alice` account and the transaction is authorized by the `active` key of the `alice` account.
+For example, the command below buys for account `bob` 0.1 XYZ worth of RAM at the current market RAM price. The cost for the RAM and the execution of this transaction is covered by the `alice` account and the transaction is authorized by the `active` key of the `alice` account.
```

Comment:
Before the final merge, we will have to replace `XYZ` with the actual Vaulta token symbol.